tests/director/api/tenants_api_test.go

```diff
@@ -28,12 +28,25 @@ func TestQueryTenants(t *testing.T) {
 	//THEN
 	t.Log("Check if tenants were received")
 
-	for _, tenant := range defaultTenants {
-		assert.Contains(t, output, tenant)
-	}
+	assertTenants(t, defaultTenants, output)
 	saveExample(t, getTenantsRequest.Query(), "query tenants")
 }
 
+func assertTenants(t *testing.T, in []*graphql.Tenant, actual []*graphql.Tenant) {
+	for _, inTnt := range in {
+		found := false
+		for _, actTnt := range actual {
+			if inTnt.ID != actTnt.ID {
+				continue
+			}
+			found = true
+
+			assert.Equal(t, inTnt.Name, actTnt.Name)
```

Comment:
I didn't check the `inUse` field value, as it could change when we ran tests in different order. I see two options here:
1. not checking the `inUse` value at all 
1. in this test create an App in one of the tenants and check if the value is true (it could be already beause of other tests, but we need to make sure the LabelDefinition is created)

I went for the option No. 1, but of course feel free to disagree with me 🙂

Reply:
What do you think about create 2 additional totally seperate tenants and for one, create runtime or application?

Reply:
Well, I see also option No. 3, adding a new tenant just for that specific use case. But I believe that would be an overkill, so I'm fine with option No. 1

Reply:
That's good idea, I will create 2 tenants (initialized and not initialized), and then I will use them for tests.